device/device.go

```diff
@@ -508,7 +523,7 @@ func (device *Device) BindUpdate() error {
 	device.queue.decryption.wg.Add(len(recvFns)) // each RoutineReceiveIncoming goroutine writes to device.queue.decryption
 	device.queue.handshake.wg.Add(len(recvFns))  // each RoutineReceiveIncoming goroutine writes to device.queue.handshake
 	for _, fn := range recvFns {
-		go device.RoutineReceiveIncoming(fn)
+		go device.RoutineReceiveIncoming(netc.bind.BatchSize(), fn)
```

Comment:
Nother const accessor thing I don't know what the convention is.

Reply:
Yup, not super critical, but I popped into a variable anyway.